tests/baselines/reference/es6ExportClauseWithoutModuleSpecifierInEs5.js

```diff
@@ -37,13 +37,13 @@ exports.x = 10;
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 var server_1 = require("./server");
-exports.c = server_1.c;
+Object.defineProperty(exports, "c", { enumerable: true, get: () => server_1.c });
 var server_2 = require("./server");
-exports.c2 = server_2.c;
+Object.defineProperty(exports, "c2", { enumerable: true, get: () => server_2.c });
 var server_3 = require("./server");
-exports.instantiatedModule = server_3.m;
+Object.defineProperty(exports, "instantiatedModule", { enumerable: true, get: () => server_3.m });
 var server_4 = require("./server");
-exports.x = server_4.x;
+Object.defineProperty(exports, "x", { enumerable: true, get: () => server_4.x });

```

Comment:
If you do the hoisting it might be possible to write these with a single `Object.defineProperties` call, but that needs some state bookkeeping.

Reply:
As mentioned above, I think the hoisting should be a separate PR.  This is something that could be considered if the bookkeeping doesn't complicate things too much.  I will note that Babel does _not_ do this, but that doesn't mean TypeScript couldn't.